examples/intro/inducing_path.py

```diff
@@ -0,0 +1,111 @@
+"""
+======================================================
+An introduction to Inducing Paths and how to find them
+======================================================
+
+A path p is called an ``inducing path`` relateve to <L,S>
+on an ancestral graph, where every non-endpoint vertex on p
+is either in L or a collider, and every collider on p is an ancestor
+of either X , Y or a member of S.
```

Comment:
More details on inducing paths can be found at :footcite:`Zhang2008`.

Then add the .. footbibliography:: at the bottom, so the references render.